chartjs/chartjs.d.ts

```diff
@@ -0,0 +1,381 @@
+// Type definitions for Chart.js
+// Project: https://github.com/nnnick/Chart.js
```

Comment:
Is this right? Which do you create this definition for `chart.js` or `chartjs` ?